### DISCUSSION TITLE
post/ifkakao2024-devrel/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#107] [Announcements]
# post/ifkakao2024-devrel/

카카오페이는 if(kakao)를 어떻게 준비했을까? Developer Reltaions와 알아보자!

https://tech.kakaopay.com/post/ifkakao2024-devrel/

**Comment by psh5487** [comment]
개발자 컨퍼런스 준비 정말 고생 많이하셨네요!
카카오페이 ifkakao 퀄리티가 좋은 이유가 다 있네요👍
